instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,33 @@
+
+import React from 'react';
+// import Images from '../../Images/favicon.png';
+
+import './SearchBar.css';
+const SearchBar = props => {
+    return (
+  
+      <div className="search-bar-wrapper">
+        <div className="social">
+        <i className="fa fa-instagram" />
+        </div>
+        <div>
+        <input type="text" placeholder="Search" onKeyDown={props.searchPosts} />
```

Comment:
oh sweet - love this onKeyDown method. I believe you can also use onChange here for the same effect